src/creatures/players/player.hpp

```diff
@@ -760,7 +770,7 @@ class Player final : public Creature, public Cylinder, public Bankable {
 	Skulls_t getSkull() const override;
 	Skulls_t getSkullClient(const std::shared_ptr<Creature> &creature) override;
 	int64_t getSkullTicks() const;
-	void setSkullTicks(int64_t ticks);
+	void setSkullTicks(int64_t ticks) const;
```

Comment:
Need to revert this too.
```suggestion
	void setSkullTicks(int64_t ticks);
```